smooth_pose_traj/CMakeLists.txt

```diff
@@ -30,7 +30,8 @@ generate_messages(
 catkin_package(
```

Comment:
Why are there now two binary files in libgeodesic? Also the make file is now deleted?

Reply:
> Why are there now two binary files in libgeodesic?

I assumed they have always been there and are generated from running make.

Reply:
> Also the make file is now deleted?

I assumed now that it is leveraging cmake that it is no longer needed. Is there a reason to keep it now that it is using cmake?